dist/unshimmed-common-from-spark301.txt

```diff
@@ -19,6 +19,7 @@ com/nvidia/spark/rapids/ShimLoader*
 com/nvidia/spark/rapids/ShimVersion*
 com/nvidia/spark/rapids/SparkShimServiceProvider*
 com/nvidia/spark/rapids/SparkShimVersion*
+com/nvidia/spark/rapids/DatabricksShimVersion*
```

Comment:
Nit: This file is lexicographically ordered, would be good to keep that precedent.